snapcraft/templates/ros2/snap/snapcraft.yaml.j2

```diff
@@ -0,0 +1,37 @@
+# The name of the snap.
+# Use 'snapcraft register <name>' to register the name on the store.
+name: {{ name }}
+# Just for humans, typically '1.2+git' or '1.3.2'
+version: "0.0.1"
+summary: Single-line elevator pitch for your amazing snap # 79 char long summary
+description: |
+  This is {{ name }}'s description. You have a paragraph or two to tell the
+  most important story about your snap. Keep it under 100 words though,
+  so that it looks good in the snap store.
+
+# The base snap is the runtime environment for this snap.
+# Bases have a 1:1 correspondence with ROS 2 LTS distributions.
```

Comment:
If I understand this, users can confidently map the number in the core to the ROS 2 LTS distro? I think we could clarify this further with an example.

Reply:
Added in https://github.com/canonical/snapcraft/pull/5516/commits/b95262af0cb529e4f9e3516342628a43423b3d03.

Reply:
Done in https://github.com/canonical/snapcraft/pull/5516/commits/690df248e8e39914c27de5f973cb19f34bd522c3